templates.json

```diff
@@ -1,8 +1,8 @@
 [
   {"template_name":  "mnist-training", "workflow_name":  "mnist_workflow_cpu"},
   {"template_name":  "mnist-training", "workflow_name": "mnist_workflow_gpu"},
-  {"template_name": "simple-example", "workflow_name": "wf"},
+  {"template_name": "basic-template-dockerfile", "workflow_name": "wf"},
   {"template_name":  "wine-classification", "workflow_name":  "training_workflow"},
   {"template_name":  "basic-template-imagespec", "workflow_name":  "wf"},
-  {"template_name":  "hello-world", "workflow_name": "hello_world_wf"}
+  {"template_name":  "hello_world", "workflow_name": "hello_world_wf"}
```

Comment:
Is there any reason to use underscore here? I ask this because other template names are using hyphen

Reply:
Looks like a typo, I can fix that, thanks!